pkg/envoy/cds/response_test.go

```diff
@@ -399,11 +408,16 @@ func TestNewResponseGetLocalServiceClusterError(t *testing.T) {
 	tassert.Nil(t, err)
 
 	ctrl := gomock.NewController(t)
+	mockKubeController := k8s.NewMockController(ctrl)
+	cfg := configurator.NewMockConfigurator(ctrl)
 	meshCatalog := catalog.NewMockMeshCataloger(ctrl)
```

Comment:
nit: initialize the mocks at the start of the function so it's easy to know the test is leveraging mocks vs fakes without needing to look through the business logic.